src/baseConfig.js

```diff
@@ -11,6 +11,8 @@ export class BaseConfig {
 
   constructor() {
     this._current = {
+      endpoint: null, // Endpoint to use for auth requests (local only). Null uses HttpClient.
```

Comment:
I've [added it](https://github.com/RWOverdijk/aurelia-auth/commit/58e9fa6d4753d28a656fc3e093d3e3205bb05159). This makes it clearer for weary travelers :D :+1: